config/config.go

```diff
@@ -548,83 +545,34 @@ func LoadConfig(path string, mandatory bool) (*Config, error) {
 	}
 
 	k := koanf.New("_")
-	if path == envKey {
-		log.Log().Info("Loading configuration from environment.")
 
-		err := loadEnvironment(k)
-		if err != nil {
-			return nil, fmt.Errorf("can't read environment config: %w", err)
+	fs, err := os.Stat(path)
+	if fs.IsDir() {
```

Comment:
This is using `fs` without checking `err` first.

Reply:
Yeah I know one of the methods I'm currently decluttering. I'll promise to handle it in the future. 😅

Reply:
Yeah no problem just thought I'd comment in case it can help debugging :stuck_out_tongue: